cpp/cmake/cudf-config.cmake.in

```diff
@@ -1,7 +1,68 @@
 @PACKAGE_INIT@
 
+
+#[=======================================================================[
+
+Provide targets for the cudf library.
+
+Built based on the Apache Arrow columnar memory format, cuDF is a GPU DataFrame
+library for loading, joining, aggregating, filtering, and otherwise
+manipulating data.
+
+cuDF provides a pandas-like API that will be familiar to data engineers &
+data scientists, so they can use it to easily accelerate their workflows
+without going into the details of CUDA programming.
+
+
+Imported Targets
+^^^^^^^^^^^^^^^^
+
+If cudf is found, this module defines the following IMPORTED GLOBAL
+targets:
+
+ cudf::cudf             - The main cudf library.
+
+This module offers an optional testing component which defines the
+following IMPORTED GLOBAL  targets:
+
+ cudf::cudftestutil     - The main cudf testing library
+ cudf::gmock_main
+ cudf::gtest_main
```

Comment:
are `cudf::gmock` and `cudf::gtest` not `IMPORTED GLOBAL` targets as well?

Reply:
They are, I didn't know if we wanted to formally document them.

Reply:
I figure we should either document them all, or only document the `cudf::cudftestutil`

Reply:
I will document them all